test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -43,7 +44,11 @@ internal static ProjectDirectory Project
 
         protected string RazorIntermediateOutputPath => Path.Combine(IntermediateOutputPath, "Razor");
 
-        protected string TargetFramework { get; set; } = "netcoreapp2.0";
+        internal static string TargetFramework
+        {
+            get => _projectTfm.Value;
+            set => _projectTfm.Value = value;
+        }
```

Comment:
This was better before, I don't think making this magical is an improvement

Reply:
We had a couple of incorrect tests because the tfm was incorrect. It's just less likely we mess up because we forgot to set the TFM too.